main/load-tx-rlp-utils.zkasm

```diff
@@ -12,7 +12,7 @@ addBatchHashData:
 getTxBytes:
         $ => A                          :MLOAD(batchL2DataLength)
         $ => B                          :MLOAD(batchL2DataParsed)
-        A - B - C - D                   :JMPN(invalidTxRLP)
+        A - B - C - D                   :JMPN(invalidTxRLP) ; TODO CHECK JUMPN only checks 64 bits
```

Comment:
```suggestion
        A - B - C - D                   :JMPN(invalidTxRLP) ; TODO CHECK JUMPN only checks 32 bits
```

Reply:
We could check that `A` register is below 32 bits and then perform the `JMPN` instruction.
`A` contains `batchL2DataLength` which comes from the input, the other regusters are intenrally computed.
I think this is probably not needed since we are checking that `batchL2DataLength` is equal to `batchHashPos` in https://github.com/0xPolygonHermez/zkevm-rom/blob/main/main/main.zkasm#L142
and the `ASSERT` instructions checks all `opX` fields elements.
Therefore, I will just remove the comment